es/CHANGELOG.md

```diff
@@ -54,16 +54,16 @@
 
 22 de agosto de 2019: 🌠 el tercer borrador se centra en un mejor texto y tiene en cuenta los aportes de la comunidad.
 
-* Con nuevas colaboradoraciones, la lista de autoras/es es nueva.
+* Con nuevas colaboraciones, la lista de autoras/es es nueva.
 * Todos los enlaces son ahora HTTPS.
 * Corrección general, aclaración de la redacción y eliminación de errores tipográficos.
 * Criterios actualizados:
-  * Requisito de reutilización en diferentes contextos
-  * Recomendación para el versionado explícito
-  * Recomendación para el desarrollo multipartito
-  * Recomendación para las cabeceras de licencia en los archivos
-  * Recomendación para la notificación de vulnerabilidades
-  * Recomendación para la documentación explícita de la gobernanza
+  * Requisito de reutilización en diferentes contextos.
+  * Recomendación para el versionado explícito.
+  * Recomendación para el desarrollo multipartito.
+  * Recomendación para las cabeceras de licencia en los archivos.
+  * Recomendación para la notificación de vulnerabilidades.
+  * Recomendación para la documentación explícita de la gobernanza.
```

Comment:
English version of the Standard doesn't have full stops, so I wouldn't integrate this suggestion at this time.